WSL/enterprise-current.md

```diff
@@ -0,0 +1,50 @@
+---
+title: WSL for Enterprise
+description: Resources and instructions on how to best use the WSL in an Enterprise environment.
+keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, ubuntu, debian, suse, windows 10, enterprise, deployment, offline, packaging, store, distribution, installation, install
+ms.date: 09/04/2018
+ms.topic: article
+ms.assetid: 7afaeacf-435a-4e58-bff0-a9f0d75b8a51
+ms.custom: seodec18
+---
+
+# Windows Subsystem for Linux for Enterprise
+
+## Creating your WSL image
+
+Once WSL is installed on your machine, you should [choose a Linux distribution](https://docs.microsoft.com/windows/wsl/install-win10#step-6---install-your-linux-distribution-of-choice) to run from the Windows store.
+
+## Exporting your custom WSL image
+
+Export your custom WSL image by running wsl --export `<Distro> <FileName>`, which will wrap your image in a tar file and make it ready for distribution on to other machines.
+
+## Distributing your WSL image
+
+Distribute the WSL image from a share or storage device by running wsl --import `<Distro> <InstallLocation> <FileName>`, which will import the specified tar file as a new distribution.
+
+## Frequently asked questions
+
+### Are there other methods to control different aspects of WSL for users in my Enterprise?
+
```

Comment:
This feels like it should be it's own section rather than an FAQ... I recommend moving it to a section called:

## Enterprise security control options

Currently, WSL offers limited control mechanisms in regard to modifying the user experience in an Enterprise scenario. Enterprise features continue in development however, below are the areas of supportted and unsupported features. To request a new feature not covered in this list, [file an issue in our GitHub repo](https://github.com/microsoft/WSL/issues?q=is%3Aissue+is%3Aopen+enterprise).